modules/internal/ChapelRange.chpl

```diff
@@ -1062,12 +1062,83 @@ module ChapelRange {
   
       newStride = if this.stride > 0 then lcm else -lcm;
     }
+
+    //
+    // These are mixed int/uint min/max functions that return a value
+    // matching the first argument's type.  They are written knowing
+    // that they will be called in an intersection context and set
+    // this variable if the intersection will be guaranteed to be
+    // empty (e.g., if all values of one range are outside of the
+    // expressible range of the other.
+    //
+    var emptyIntersection: bool;
```

Comment:
Maybe I'm missing something, but to me it looks like this isn't really used. Was it just for debugging purposes?